docs/core/deploying/index.md

```diff
@@ -60,6 +64,28 @@ It also has a number of disadvantages:
 
 - Deploying numerous self-contained .NET Core apps to a system can consume significant amounts of disk space, since each app duplicates .NET Core files.
 
+## Framework-dependent executables (FDE)
+
+Starting with .NET Core 2.2, you deploy your app as an FDE, and any required thrid-party dependencies. You don't have to deploy .NET Core, since your app will use the version of .NET Core that's installed on the target system.
+
+### Why deploy a framework-dependent executable?
+
+Deploying an FDE has a number of advantages:
+
+- The size of your deployment package is small. You only deploy your app and its dependencies, not .NET Core itself.
+
+- Multiple apps use the same .NET Core installation, which reduces both disk space and memory usage on host systems.
+
+- Your app can be run by calling the published executable without invoking the `dotnet` utility directly.
+
+There are also a few disadvantages:
+
+- Your app can run only if the version of .NET Core that you target, or a later version, is already installed on the host system.
```

Comment:
that you target -> that you're targetting on your app or project?

later version -> later patch version? I imagine it behaves like the FDD 

Also, we need to update the following section since we now also support exes: https://docs.microsoft.com/en-us/dotnet/core/versions/selection#framework-dependent-apps-roll-forward

and maybe link to this from here?